stream/src/public_key/handshake.rs

```diff
@@ -27,6 +27,8 @@ pub struct Info<P: PublicKey> {
 
     /// Timestamp of the handshake (in epoch milliseconds).
     timestamp: u64,
+    /// c.f. [`Self::with_tag_over`]
+    continuation_tag: Option<<Sha256 as Hasher>::Digest>,
```

Comment:
nit: you might be able to just use `sha256::Digest` as well if you `use commonware_cryptography::sha256`